domain/application/service/package_test.go

```diff
@@ -47,6 +48,7 @@ type baseSuite struct {
 	agentVersionGetter        *MockAgentVersionGetter
 	provider                  *MockProvider
 	supportedFeaturesProvider *MockSupportedFeatureProvider
+	k8sBroker                 *MockKubernetesBroker
```

Comment:
Should we not be changing the name k8s, kubernetes to just CAASBroker? We have the provider layer for a reason seems weird to bleed this naming out.

Reply:
We long ago gave up the pretext of being able to support anything other than k8s. There's a number of other places where we use this naming convention. But easy to change here.